app/assets/stylesheets/vendor/devices.scss

```diff
@@ -0,0 +1,152 @@
+@mixin border-radius($radius) {
+  -webkit-border-radius: $radius;
+  border-radius: $radius;
+}
+
+.device {
+  display: inline-block;
+  position: relative;
+  -webkit-box-sizing: content-box;
+  -moz-box-sizing: content-box;
+  box-sizing: content-box;
+
+  .screen {
+    width: 100%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered -webkit-box-shadow, box-shadow, color, display, height, position, text-align, width, z-index